README.md

```diff
@@ -480,18 +480,19 @@ Similarly, cross-document navigations initiated from other windows are not somet
 
 As for `document.open()`, it is a terrible legacy API with lots of strange side effects, which makes supporting it not worth the implementation cost. Modern sites which use the new navigation API should never be using `document.open()`.
 
-Second, the following navigations **cannot be canceled** using `event.preventDefault()`, and as such will have `event.cancelable` equal to false:
-
+Second, **traversals have special restrictions on cancelling the navigation** via `event.preventDefault()`. Traversals are:
 - User-initiated traversals via the browser's back/forward buttons (either same- or cross-document)
 - Programmatic traversals via `history.back()`/`history.forward()`/`history.go()`
-- Programmatic traversals via `navigation.back()`/`navigation.forward()`/`navigation.go()`
+- Programmatic traversals via `navigation.back()`/`navigation.forward()`/`navigation.traverseTo()`
 
-We would like to make these cancelable in the future. However, we need to take care when doing so:
+Traversals may only be cancelled (and `event.cancelable` will be equal to true) if:
+- The navigate event is firing in the top window
+- The traversal is same-origin
+- The traversal was not user-initiated, or there is a consumbable user activation in the current window.
```

Comment:
I think we should write the explainer toward the desired end state, of "consumable sticky activation". This will require a few sentences or a paragraph of extra explanation, but it seems better than putting something in the explainer we don't plan to ship.

Reply:
Added a paragraph